plugin/emacs/test/quicklintjs-test.el

```diff
@@ -13,7 +13,7 @@
   (setq package-user-dir cache-dir-name
         package-check-signature nil)
   (add-to-list 'package-archives
-               '("MELPA" . "https://stable.melpa.org/packages/"))
+               '("melpa" . "https://melpa.org/packages/") t)
```

Comment:
Which was the problem: capitalization of the first string, `stable.`, or ` t`?

Reply:
I guess this wasn't the issue, the issue is probably because of older emacs version which has some emacs-gpg issues.